src/routes/plans/+page.svelte

```diff
@@ -267,15 +272,15 @@
       }));
       await effects.createPlanTags(newPlanTags, newPlan, user);
       newPlan.tags = planTags.map(tag => ({ tag }));
-      plans = [...plans, newPlan];
+      plans.updateValue(() => [...$plans, newPlan]);
```

Comment:
Small nit pick. The `updateValue` function passes the current value (i.e. `plans`) as the argument to the callback. It's exactly the same exact thing as what you have here, it just feels a little more in line with how it was intended to be used if we used the current value. I'll defer to you to make the final call on that though since it's a moot point.

Reply:
Good catch, agreed.